robyn/argument_parser.py

```diff
@@ -1,13 +1,35 @@
 import argparse
 
+
 class ArgumentParser(argparse.ArgumentParser):
     def __init__(self):
-        self.parser = argparse.ArgumentParser(description="Robyn, a fast async web framework with a rust runtime.")
-        self.parser.add_argument('--processes', type=int, default=1, required=False)
-        self.parser.add_argument('--workers', type=int, default=1, required=False)
-        self.parser.add_argument('--dev', default=False, type=lambda x: (str(x).lower() == 'true'))
+        self.parser = argparse.ArgumentParser(
+            description="Robyn, a fast async web framework with a rust runtime."
+        )
+        self.parser.add_argument(
+            "--processes",
+            type=int,
+            default=1,
+            required=False,
+            help="Choose the number of processes. [Default: 1]",
+        )
+        self.parser.add_argument(
+            "--workers",
+            type=int,
+            default=1,
+            required=False,
+            help="Choose the number of workers. [Default: 1]",
+        )
+        self.parser.add_argument(
+            "--dev",
+            dest="dev",
+            action="store_true",
```

Comment:
Thank you for contributing. But can you tell me what these flags do?

Reply:
Yes! So, the change here is removing the `dev` in the arguments and add in the options. The reason is that no one will ever do `--dev=false` as it's the default.

This change just makes `--dev` standalone possible. The `action` argument on the highlighted above makes it store a `True` value on the `dev` variable.

Reply:
Oh, alright. Makes sense. Thank you! :D